crates/sovereign-sdk/module-system/sov-state/src/zk_storage.rs

```diff
@@ -93,6 +93,7 @@ where
                 let value_bytes = value
                     .map(|v| RefCount::try_unwrap(v.value).unwrap_or_else(|arc| (*arc).clone()));
 
+                // Seems like we can get rid of the extra clone here
```

Comment:
I am working on this right now as it will also simplify the api I will have for the Cache I return for reuse. Can remove